recipe/meta.yaml

```diff
@@ -51,6 +51,7 @@ requirements:
 test:
   commands:
     - python -c "import rdkit; assert rdkit.__version__ == '{{ version }}'"
+    - if not exist %LIBRARY_INC%\\rdkit\\Catalogs\\Catalog.h exit 1   # [win]
```

Comment:
Add more existence tests to ensure that all the expected outputs (types) are being included in the new combined package.